rust/processor/src/db/parquet/models/token_v2_models/v2_token_ownerships.rs

```diff
@@ -507,3 +508,108 @@ impl TokenOwnershipV2 {
         Ok(None)
     }
 }
+
+// // Copyright © Aptos Foundation
```

Comment:
TODO: Didn't update this yet, have to decide how to handle lookups

Reply:
this is handled in pt2 pr

Reply:
Commented out portion is deleted in [next PR](https://github.com/aptos-labs/aptos-indexer-processors/blob/979c1b85f7a021508d818c027d48fe92a04ab7f3/rust/processor/src/db/postgres/models/token_v2_models/v2_token_ownerships.rs). Leaving in to prevent having to remerge here.